### ISSUE TITLE
Clean up TCK release notes for Jakarta Standard Tag Library

### ISSUE THREAD

**Issue by pnicolucci** [eclipse-ee4j/jstl-api#132]
See discussion here :https://www.eclipse.org/lists/jakarta.ee-spec/msg01206.html

**Comment by pnicolucci** [comment]
The current set of docs have made the corrections for Jakarta EE9. All we need to do for this issue is ensure that the index and ReleaseNotes are property updated for Jakarta EE10: https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/install/jstl/docs

**Comment by pnicolucci** [comment]
@scottmarlow is it the Jakarta Tags team responsibility to update these in the TCK or does the TCK project team take care of this?

**Comment by scottmarlow** [comment]
@pnicolucci it doesn't matter much to me as to whom makes the changes as long as they are complete before Jakarta Tags is released as final (as we cannot make further Tags TCK changes to doc after that).  I would be happy with doc changes from any contributor, regardless of whether they are committors on Jakarta Tags or Platform TCK projects.

FYI the previous update for EE 9.1 was via https://github.com/eclipse-ee4j/jakartaee-tck/commit/3b179ec7e60ea6751cd999a80d2b0cdca34b729b#diff-9a5de9445b8718b8b26ab9e7ff3ac55a3d4a71ec3ccd88b0576d864a028ec33d

**Comment by scottmarlow** [comment]
@pnicolucci I'm giving the update a shot.  I'll create a draft pr with the results.  Do you plan to run the Tags TCK yourself?  Or will someone else run that?  I would like to ask which operating system will be used for the testing.  The current Tags TCK release notes includes:
```
- Jakarta EE Standard Tag Library TCK tests run on the following platforms: 
- Alpine Linux v3.12- 
- CentOS Linux 7
```
I will need to know what to change ^ to.

For reference, I just installed `bluegriffin` to edit the Tags TCK html files.

CC @edbratt

**Comment by pnicolucci** [comment]
Hey @scottmarlow , what are our options for running the TCK. I believe
@arjantijms  did this in the past for this project. @arjantijms any input here?

For the platforms do we just need to list whatever platform is actually used when the tests are executed? If that's the case
then it just depends on who executes the TCK once we have something available for testing.

I believe previously for Tags 2.0 we used Glassfish as the compatible implemented so my plan would be to do the same this go around for Tags 3.0.

**Comment by scottmarlow** [comment]
> Hey @scottmarlow , what are our options for running the TCK. I believe @arjantijms did this in the past for this project. @arjantijms any input here?

https://jakarta.ee/specifications/tags/2.0/ shows that https://github.com/eclipse-ee4j/glassfish/releases/download/6.0.0-M3-2020-10-04/glassfish-6.0.0-M3-2020-10-04.zip was used for verifying Tags 2.0.  I imagine that the first available Tags 3.0 implementation could be used to validate the Tags 3.0 release.  As to the how or who can run that, IMO, whomever that is, they need to push the TCK results to a publicly available location and identify the publicly available Tags implementation (and version) and include the implementation download link (as per https://jakarta.ee/committees/specification/tckprocess/).

As you proceed through the Tags ballot process, ^ is good to discuss with your mentor as well.

> 
> For the platforms do we just need to list whatever platform is actually used when the tests are executed? If that's the case then it just depends on who executes the TCK once we have something available for testing.

Makes sense.

> 
> I believe previously for Tags 2.0 we used Glassfish as the compatible implemented so my plan would be to do the same this go around for Tags 3.0.

I think that would need to be a milestone release build of the Tags 3.0 compatible implementation, GlassFish 7.0 seems like a fine choice but it really could be any (open source) Tags 3.0 compatible implementation that is publicly available in a milestone release form.

**Comment by volosied** [comment]
The issues in the original email thread were addressed by Scott's PR.

I'll close this issue, but open a new issue about running the TCK on Glassfish once we have a milestone release.
